### ISSUE TITLE
put Auto-downloaded maps in low priority

### ISSUE THREAD

**Issue by ashdnazg** [Spring-Chobby/Chobby#363]
When a user logs into Chobby for the first time the curated maps are added into the download queue even if they entered a room with a non-curated map.
That means waiting quite a long time for all of them to download, instead of downloading the current map first.

**Comment by GoogleFrog** [comment]
Is low priority something which is supported in the engine?

**Comment by ashdnazg** [comment]
No, but the fact that all featured maps are insta-added to the queue may not be the smartest action.

**Comment by Licho1** [comment]
Another option is for wrapper to get those maps (delay first start of cobby)

2016-10-22 18:17 GMT+02:00 Eshed <notifications@github.com>:

**Comment by gajop** [comment]
I think the engine shouldn't support queue priority, but rather allow reordering of items in the queue. Currently reordering can be implemented with abort & readd.

There's an idea to remove queues from the engine altogether, but we should consider the benefits we could have with an engine implementation (e.g. *potential* ability to verify if downloads exist beforehand, give an estimate of the total download size, etc.)
